### PR TITLE
CORE-642: In Core Generic, add genWalletAddTransfer()

### PR DESCRIPTION
**Pull request by EBGToo**
@carlcherry Can you explore this change with your Ripple BDB?

### CODE REVIEW

File: generic/BRGenericRipple.c [hunk 1]
```diff
@@ -73,8 +71,6 @@ genericRippleAccountSignTransferWithKey (BRGenericAccountRef account,
     // Get the transaction pointer from this transfer
     BRRippleTransaction transaction = rippleTransferGetTransaction ((BRRippleTransfer) transfer);
     if (transaction) {
-        // Hard code the sequence to 7
-        rippleAccountSetSequence ((BRRippleAccount) account, 7);
 //        rippleAccountSignTransaction(account, transaction, seed);
         assert (0);
```
Comment (carlcherry) [inline generic/BRGenericRipple.c]: Should we just remove the function instead?

Comment (EBGToo) [reply generic/BRGenericRipple.c]: We could.

**Comment by EBGToo** [comment]
Also, if a Ripple Wallet now holds all the transfers, can the Ripple Wallet compute a correct balance?  If so, the cryptoWalletUpdateBalance() function, calling cryptoTransferGetAmountDirectedNet(), is no longer needed.

**Comment by EBGToo** [comment]
@carlcherry I updated the Ripple wallet balance to include the transfer fee.

**Pull request merged**